library/struct.po

```diff
@@ -223,36 +291,54 @@ msgid ""
 "switchable endianness (bi-endian). Use ``sys.byteorder`` to check the "
 "endianness of your system."
 msgstr ""
+"Le boutisme natif est gros-boutiste ou petit-boutiste, en fonction de la "
+"machine sur laquelle s'exécute le programme. Par exemple, les Intel x86 et "
+"les AMD64 (x86-64) sont petit-boutistes ; les Motorola 68000 et les *PowerPC "
+"G5* sont gros-boutistes ; les ARM et les Intel Itanium peuvent basculer de "
+"boutisme. Utilisez ``sys.byteordre`` pour vérifier le boutisme de votre "
```

Comment:
```suggestion
"boutismes. Utilisez ``sys.byteordre`` pour vérifier le boutisme de votre "
```
Ou « changer de boutisme », mais « basculer de boutisme » me dérange un peu.

Reply:
Le « to switch » original m'a fait traduire comme ça. Mais je prends « changer ».